e2e/playwright/search/src/tests/search-input.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test } from '@alfresco/playwright-shared';
+
+test.describe('Search - Input', () => {
+  const username = `user1-${Utils.random()}`;
+
+  test.beforeAll(async () => {
+    try {
+      const apiClientFactory = new ApiClientFactory();
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+    } catch (error) {
+      console.error(`beforeAll failed: ${error}`);
+    }
+  });
+
+  test.beforeEach(async ({ loginPage }) => {
+    try {
+      await loginPage.loginUser({ username, password: username }, { withNavigation: true, waitForLoading: true });
+    } catch (error) {
+      console.error(`beforeEach failed: ${error}`);
+    }
+  });
+
+  test('[C289847] Search icon is displayed in toolbar and clicking on it displays search input container', async ({ personalFiles, searchPage }) => {
+    await personalFiles.acaHeader.searchButton.click();
+    await searchPage.searchInput.searchButton.click();
+
+    expect(await searchPage.searchOverlay.searchInputControl).toBeVisible();
+  });
+
+  test('[C289848] Search options are displayed when clicking in the search input', async ({ personalFiles, searchPage }) => {
+    await personalFiles.acaHeader.searchButton.click();
+    await searchPage.searchInput.searchButton.click();
+
+    expect(await searchPage.searchOverlay.searchOptions).toBeVisible();
```

Comment:
I can see many warnings in whole file for missing await because some of the functions like toBeVisible, toBeEnabled etc. return Promise. 
<img width="1114" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/7c9c60e5-f8df-40e6-a82f-b8f5be687a5f">